src/main/kotlin/blackJack/domain/Player.kt

```diff
@@ -0,0 +1,38 @@
+package blackJack.domain
+
+import blackJack.domain.Status.HIT
+import blackJack.error.ErrorMessage
+
+class Player(val name: String, val cards: Cards = Cards(mutableListOf()), var status: Status = HIT) {
+
+    init {
+        require(name.isNotEmpty()) { ErrorMessage.EMPTY_NAME.message }
+    }
+
+    fun addCard(cardDeck: Cards) {
+        Status.validationAddCard(status)
+        val card = cardDeck.drawCard()
+        cards.addCard(card)
+        status = Status.calculateStatus(cards.calculateTotalScore(), cards.cardSize)
+    }
+
+    fun receiveInitialCards(initialCards: Cards) {
+        cards.addCard(initialCards.drawCard())
+        cards.addCard(initialCards.drawCard())
```

Comment:
```suggestion
      cards.addAll(initialCards)
```
addAll 함수를 만든다면, 두번 실행시켜줄 필요가 없지 않을까요~?